src/main/java/me/totalfreedom/totalfreedommod/command/Command_list.java

```diff
@@ -68,31 +54,54 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
             listFilter = ListFilter.PLAYERS;
         }
 
+        if (listFilter == ListFilter.VANISHED_ADMINS && !isAdmin(playerSender))
+        {
+            msg("You can't view vanished admins.");
```

Comment:
I probably wouldnt even list this, I'd just make it return an unkown command with the syntax.